src/index.js

```diff
@@ -1,5 +1,8 @@
+// @flow
 /**
  * React Flip Move
  * (c) 2016-present Joshua Comeau
  */
-module.exports = require('./FlipMove');
+import FlipMove from './FlipMove';
+
+export default FlipMove;
```

Comment:
`export default from '...'` is on [stage 1](https://github.com/leebyron/ecmascript-export-default-from) yet, and Flow doesn't understand it.

And Flow loses type information when you use commonJs imports with `require`.

This:
```js
import FlipMove from './FlipMove'
module.exports = FlipMove
```
seems to work though, but it shouldn't be really needed, because you use `babel-plugin-add-module-exports`anyway, and `export default FlipMove` gets compiled to
```js
exports.default = _FlipMove2.default;
module.exports = exports['default'];
```
given that there's no named exports.